cwl/src/test/scala/wdl4s/cwl/ExportCwlSamplesSpec.scala

```diff
@@ -2,17 +2,17 @@ package wdl4s.cwl
 
 import org.scalatest.{FlatSpec, Matchers}
 import shapeless.Coproduct
-import shapeless.syntax.singleton._
 import wdl4s.cwl.CommandLineTool.{BaseCommand, StringOrExpression}
 import wdl4s.cwl.WorkflowStepInput.InputSource
 
 class ExportCwlSamplesSpec extends FlatSpec with Matchers {
-  def assertCorrectJson(cwl: Cwl, expectedYaml: String) = CwlCodecs.cwlToYaml(cwl) shouldBe expectedYaml
+  //TODO: Re-implement ability to write CWL to Yaml
```

Comment:
Ticket > `TODO`

Reply:
agree, changing to a link to [issue](https://github.com/broadinstitute/wdl4s/issues/216)